roles/foreman_installer/tasks/main.yml

```
@@ -0,0 +1,10 @@
+---
+- name: 'Run installer'
+  shell: >
```

```suggestion
  command: >
```

There is no need for a `shell` here

---

I wanted to protect the foreman-installer's back. (Friday dad jokes, you're welcome)